.changeset/polite-tigers-end.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": minor
+---
+
+Add Pipelines commands to Wrangler
```

Comment:
Could you modify this to follow the changeset format? Additionally, since it's a big feature a couple examples of how to use the commands would be nice!